internal/pkg/deploy/cloudformation/task.go

```diff
@@ -48,3 +52,82 @@ func (cf CloudFormation) DeployTask(input *deploy.CreateTaskResourcesInput, opts
 
 	return nil
 }
+
+// ListTaskStacks returns all the CF stacks which represent one-off copilot tasks in a given application's environments
+func (cf CloudFormation) ListTaskStacks(appName, envName string) ([]deploy.TaskStackInfo, error) {
```

Comment:
What do you think instead of having a public function `ListStacksWithPrefix`, we create make the one that can list all the stacks with specific tag set? So that this function can be reused by listing tasks running in default cluster as well.